src/reducers/MapReducer.js

```diff
@@ -47,10 +54,79 @@ const INITIAL_STATE = {
     },
     mapLoaded: false,
     viewStyle: map.SearchOptions.MAP,
-    sites: []
+    sites: [],
+    displaySites: [],
+    filterCriteriaKeys: {accessibility: [], facilities: [], price: []}
```

Comment:
Keep track of the filter keys in their three different categories. This is important for the filtering logic as we'll see below.